### PR TITLE
462 provider page information for italian network providers

### PR DESCRIPTION
**Pull request by Mandloi2309**
## Issue
Provider markdown and logo updates

### CODE REVIEW

**Comment by mriios** [comment]
@Mandloi2309 I thought the image name was wrong but I think it's the abbrevs? Could you double check, please? Clicking on "see more models" on those providers returns no results (bad api query)

**Comment by Mandloi2309** [comment]
Ah, yes. It needs to be other way. Provider name is IRCCS-IOR so will have to fix in markdown. 
But if this works we can keep it this way

**Comment by Mandloi2309** [comment]
Added some more providers from Italian network

**Comment by Mandloi2309** [comment]
All providers are added and can be merged if all okay

File: public/static/providers/irccs-ior.md [hunk 1]
```diff
@@ -1,5 +1,16 @@
 ---
 name: Istituto Ortopedico Rizzoli
 abbreviation: IOR-IRCCS
```
Comment (mriios) [inline public/static/providers/irccs-ior.md:3]: @Mandloi2309  Is this a mistake?

Comment (Mandloi2309) [reply public/static/providers/irccs-ior.md:3]: Yes, great eye of observation!

Comment (Mandloi2309) [reply public/static/providers/irccs-ior.md]: made the change

**Comment by mriios** [comment]
Looks good! Merging

**Pull request merged**